### DISCUSSION TITLE
Symfony Docs barebones issues

### DISCUSSION THREAD

**Discussion by gnito-org** [symfony/symfony#43083] [Q&A]
When there is an almost empty issue in symfony/docs such as this one at https://github.com/symfony/symfony-docs/issues/15793, does that mean the issue is asking that someone write the documentation for the linked symfony/symfony pull request?

**Comment by wouterj** [comment]
These issues are generated automatically when a code PR is merged without docs. So yes, these are undocumented.

If you're new to the docs, it's best to first leave a comment to ask if something needs docs. Sometimes, a feature is only internal or does not need to be documented. A doc team member can tell you whether we want it documented.

**Comment by gnito-org** [comment]
Thanks for the quick answer.

To be honest, that sounds a bit cumbersome.

Would it make sense to have a contribution rule that says, "if docs are NOT needed, then explicitly state that in the PR"?

That way one just needs to check the linked code PR to see if the docs issue requires contribution or not, instead of asking the question for each and every skeleton docs issue.
